### PR TITLE
Add brief description tooltips to EditorResourcePicker

### PR DESCRIPTION
**Pull request by KoBeWi**
https://github.com/godotengine/godot/assets/2223172/652351d9-e633-4021-a564-49746607aede

The implementation is very simple, so why not.

### CODE REVIEW

**Review by Mickeon — COMMENTED** [review]
Surprisingly simple. As discussed in RocketChat, it's better than not having the tooltip at all.
Note that the tooltip does not support BBCode formatting at all, but it's not an issue exclusive to the resource picker.

**Review by Mickeon — APPROVED** [review]

**Comment by JekSun97** [comment]
the value is taken from the documentation, as I understand it, the translation will not be applied?

**Comment by Mickeon** [comment]
They would, assuming the brief description is localized by the translators, which unfortunately is not a given.

File: editor/editor_resource_picker.cpp [hunk 1]
```diff
@@ -504,6 +505,11 @@ void EditorResourcePicker::set_create_options(Object *p_menu_node) {
 			int id = TYPE_BASE_ID + idx;
 			edit_menu->add_icon_item(icon, vformat(TTR("New %s"), t), id);
 
+			HashMap<String, DocData::ClassDoc>::Iterator class_doc = EditorHelp::get_doc_data()->class_list.find(t);
+			if (class_doc) {
+				edit_menu->set_item_tooltip(-1, class_doc->value.brief_description);
```
Comment (akien-mga) [inline editor/editor_resource_picker.cpp:510]: Needs to go through `DTR` for translations, no?
```suggestion
				edit_menu->set_item_tooltip(-1, DTR(class_doc->value.brief_description));
```

**Review by akien-mga — APPROVED** [review]

**Review by AThousandShips — APPROVED** [review]

**Pull request merged**